mlrun/db/httpdb.py

```diff
@@ -3264,6 +3266,41 @@ def _wait_for_background_task_to_end(bg_name):
 
         return bg_task
 
+    def get_datastore_profile(
+        self, name: str, project: str
+    ) -> Optional[mlrun.common.schemas.DatastoreProfile]:
+        project = project or config.default_project
+        path = self._path_of("projects", project, "datastore_profiles") + f"/{name}"
+
+        res = self.api_call(method="GET", path=path)
+        if res and res._content:
```

Comment:
Don't use internal members (`_content`). Instead use `res.json()` to retrieve the json contents (it also replaces the `json.loads` you're doing later). If the response is empty then you will have an empty dict, which you can check for.